solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -55,9 +109,9 @@ public float score(float[] featureVector) {
       }
 
       if (featureVector[featureIndex] <= threshold) {
-        return left.score(featureVector);
+        return (left == null ? 0f : left.score(featureVector));
```

Comment:
remove the null checks and make sure validate throws an exception instead (check all left/right explain/score functions)